docs/migration_ov_2_0/snippets/ov_api/ov_common.cpp

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#include <openvino/core/function.hpp>
+#include <openvino/runtime/runtime.hpp>
+
+int main() {
+    //! [ov_api_2_0:create_core]
+    ov::runtime::Core core;
+    //! [ov_api_2_0:create_core]
+
+    //! [ov_api_2_0:read_model]
+    std::shared_ptr<ov::Function> network = core.read_model("model.xml");
```

Comment:
should we use 'auto' in snippets?

Reply:
I don't want to use auto. THis code is used in the documentation and I would like to show all returned types.